app/models/aggregation.rb

```diff
@@ -14,8 +14,15 @@ class Aggregation < ActiveRecord::Base
   validate :aggregation, :workflow_version_present
 
   def self.scope_for(action, user, opts={})
-    return super unless (action == :show || action == :index) && !user.is_admin?
-    joins(:workflow).merge(Workflow.scope_for(:update, user, opts))
+    case
+    when action == :index && opts.has_key?(:public_workflow_ids)
```

Comment:
Trying to tap into controlled resources in RoledController for the before_action [here](https://github.com/zooniverse/Panoptes/blob/1e8b9cadfbf67c60354158fddd4d4a1980eb9696/lib/role_control/roled_controller.rb#L8)